### PR TITLE
Minor code coverage build modifications

### PR DESCRIPTION
**Pull request by YvanMokwinski**
make things standard across libraries. remove g option since it's already automatically added in the allowed modes.

### CODE REVIEW

**Review by cgmb — APPROVED** [review]
LGTM

File: install.sh [hunk 1]
```diff
@@ -543,9 +543,9 @@ case "${ID}" in
     ;;
 esac
 
-if [[ "${build_coverage}" == true ]]; then
+if [[ "${build_codecoverage}" == true ]]; then
     if [[ "${build_type}" == Release ]]; then
-        echo "Code coverage is not supported in Release mode, to enable code coverage select either Debug mode (-g | --debug) or RelWithDebInfo mode (-k | --relwithdebinfo); aborting";
+        echo "Code coverage is chosen to be disabled in Release mode, to enable code coverage select either Debug mode (-g | --debug) or RelWithDebInfo mode (-k | --relwithdebinfo); aborting";
```
Comment (tfalders) [inline install.sh:548]: I'd prefer simply "disabled" over "chosen to be disabled".

**Review by tfalders — APPROVED** [review]
LGTM!

**Review by jzuniga-amd — APPROVED** [review]

File: install.sh (hunk 1 above)
Comment (YvanMokwinski) [reply install.sh:548]: well, modify as you wish.

**Pull request merged**